### DISCUSSION TITLE
write data to SD card using berry scripting?

### DISCUSSION THREAD

**Discussion by Brightness248** [arendst/Tasmota#21748] [Q&A] [answered]
Hello community.
I make my own PCB's and my current project is about reading values from an AHT20 and MPU6050 and writing them to an SD card.
With Berry and the module `import persist` it was very easy to write to the flash. With the extension of an SD card, I can't find a way to write to the SD card with Berry. 

I have found solutions with "scripting". I don't like this kind of scripting at all. Berry is much clearer, easier to read and simply nicer (at least to me^^).

Is there currently a way to write to the SD card with Berry?

**Comment by sfromis** [comment, marked as answer]
See the `file` class in the Berry language documentation:
https://berry.readthedocs.io/en/latest/source/en/Chapter-7.html#file-class

Minimal example:
```
fh=open('/sd/sdtest.txt','w')
fh.write('sd test data')
fh.close()
```

**Reply by Brightness248** [reply]
that works.
Thank you very much.
